fiscalyear.py

```diff
@@ -815,6 +864,220 @@ def __ge__(self, other):
+            an int or int-like string
+        :raises ValueError: If fiscal_year or fiscal_day is out of range
+        """
+        fiscal_year = _check_year(fiscal_year)
+        fiscal_day = _check_fiscal_day(fiscal_year, fiscal_day)
+
+        self = super(FiscalDay, cls).__new__(cls)
+        self._fiscal_year = fiscal_year
+        self._fiscal_day = fiscal_day
+        return self
+
+    @classmethod
+    def current(cls):
+        """Alternative constructor. Returns the current FiscalDay.
+
+        :returns: A newly constructed FiscalDay object
+        :rtype: FiscalDay
+        """
+        today = FiscalDate.today()
+        return cls(today.fiscal_year, today.fiscal_day)
+
+    def __repr__(self):
+        """Convert to formal string, for repr().
+
+        >>> fd = FiscalDay(2017, 1)
+        >>> repr(fd)
+        'FiscalDay(2017,1)'
+        """
+        return '%s(%d, %d)' % (self.__class__.__name__,
+                               self._fiscal_year,
+                               self._fiscal_day)
+
+    def __str__(self):
+        """Convert to informal string, for str().
+
+        >>> fd = FiscalDay(2017, 1)
+        >>> str(fd)
+        'FY2017 FD1'
+        """
+        return 'FY%d FD%d' % (self._fiscal_year, self._fiscal_day)
+
+    # TODO: Implement __format__ so that you can print
+    # fiscal year as 17 or 2017 (%y or %Y)
+
+    def __contains__(self, item):
+        """Returns True if item in self, else False.
+
+        :param item: The item to check
+        :type item: FiscalYear, FiscalQuarter, FiscalDateTime,
+            datetime, FiscalDate, or date
+        :rtype: bool
+        """
+        if isinstance(item, FiscalDay):
+            return self == item
+        elif isinstance(item, datetime.datetime):
+            return self.start <= item <= self.end
+        elif isinstance(item, datetime.date):
+            return self.start.date() <= item <= self.end.date()
+        else:
+            raise TypeError("can't compare '%s' to '%s'" % (
+                type(self).__name__, type(item).__name__))
+
+    # Read-only field accessors
+
+    @property
+    def fiscal_year(self):
+        """:returns: The fiscal year
+        :rtype: int
+        """
+        return self._fiscal_year
+
+    @property
+    def fiscal_quarter(self):
+        """:returns: The fiscal quarter
+        :rtype: int
+        """
+        return self.start.quarter
+
+    @property
+    def fiscal_month(self):
+        """:returns: The fiscal month
+        :rtype: int
+        """
+        return self.start.fiscal_month
+
+    @property
+    def fiscal_day(self):
+        """:returns: The fiscal day
+        :rtype: int
+        """
+        return self._fiscal_day
+
+    @property
+    def start(self):
+        """:returns: Start of the fiscal day
+        :rtype: FiscalDateTime
+        """
+
+        fiscal_year = FiscalYear(self._fiscal_year)
+        days_elapsed = datetime.timedelta(days=self._fiscal_day - 1)
```

Comment:
Does this work when `self._fiscal_day == 1`?

Reply:
Zero days have elapsed on fiscal day one so it's year start plus zero on `self.fiscal_day == 1`. Do you think that's confusing?

I believe it's adequately covered by test_fiscalyear:886:
`assert a.start == fiscalyear.FiscalYear(a.fiscal_year).start`